src/xdocs/config_filters.xml

```diff
@@ -633,6 +634,161 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressionXpathFilter">
+      <subsection name="Description">
+        <p>Since Checkstyle 8.2</p>
```

Comment:
8.2 has already come and gone. Update to expected release.

Reply:
done